DataConnectors/ForgeRock_CEF.json

```diff
@@ -0,0 +1,112 @@
+{
+  "id" : "ForgeRock_Common_Audit_CEF",
+  "title" : "ForgeRock Common Audit for CEF",
+  "publisher" : "ForgeRock Inc",
+  "descriptionMarkdown" : "The ForgeRock Identity Platform provides a single common auditing framework. Extract and aggregate log data across the entire platform with common audit (CAUD) event handlers and unique IDs so that it can be tracked holistically. Open and extensible, you can leverage audit logging and reporting capabilities for integration with Microsoft Azure Sentinel via this CAUD for CEF connector.",
+  "graphQueries" : [
+    {
+      "metricName" : "Total data received",
+      "legend" : "ForgeRock",
+      "baseQuery" : "CommonSecurityLog | where DeviceVendor in (\"ForgeRock Inc\") | where DeviceProduct in (\"Trust Partner Network\")\n| where DeviceAction =~ \"FAILED\""
+    }
+  ],
+  "sampleQueries" : [
+    {
+      "description" : "Total failed login attempts to the ForgeRock Platform",
+      "query" : "CommonSecurityLog | where DeviceVendor in (\"ForgeRock Inc\") | where DeviceProduct in (\"Trust Partner Network\")\n| extend user = split(split(AdditionalExtensions, \";\", 4)[0], \"=\", 1)[0] // extract userID from  4th field in json payload\n| where strlen(user) > 2 // if not null\n| where DeviceAction =~ \"FAILED\""
```

Comment:
The sample queries are pretty complicated. The goal is not only to get data into Azure Sentinel but also in a format that enables customers to easily work with the data in different Azure Sentinel experiences. I have provided feedback on the sample data as well. Please map the values embedded in msg and other fields into relevant CEF columns. That way the queries become very simple. The queries are ultimately used in building workbooks, hunting queries, analytics etc. so having a simple way to work with the data in columnar formats is very critical.